src/IW/Db/Repo.hs

```diff
@@ -33,12 +33,12 @@ getReposByCategory = query [sql|
 -- | Insert a list of repos into the database, but update on conflict
 upsertRepos :: (WithDb env m) => [Repo] -> m ()
 upsertRepos = executeMany [sql|
-    INSERT INTO repos 
+    INSERT INTO repos
         (owner, name, descr, categories)
-    VALUES 
```

Comment:
Interesting fact, but having trailing whitespaces in SQL queries can actually decrease performance :slightly_smiling_face: Because these queries are parsed at runtime and sent via socket to DB. So, the more characters they have the slower they will be. Well, even if we have 10 trailing whitespaces on each line, we won't even notice the slowdown. It's just an interesting observation of how a style guide can affect performance

Reply:
Wow, that is an interesting fact. I had no idea whitespace could make your program slower.

Reply:
And you haven't even heard about stories where having comments in your code can make your code slower...